src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -30,7 +30,8 @@ public AuthenticationTests()
 
         public void Dispose() => _factory.Dispose();
 
-        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementationAndNotUap))] // Managed implementation connects successfully.
+        [ConditionalTheory(nameof(Helpers) + "." + nameof(Helpers.IsWindowsImplementation))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap, "Not supported on UWP")]
```

Comment:
What's our plan for HttpListener on UWP?  Is there a reason we can't just use the managed implementation?